pkg/master/controller.go

```diff
@@ -69,7 +69,8 @@ type Controller struct {
 	SystemNamespaces         []string
 	SystemNamespacesInterval time.Duration
 
-	PublicIP net.IP
+	PublicIP   net.IP
+	PublicPort int
```

Comment:
what is the difference between this and the PublicServicePort below?

Reply:
or KubernetesServiceNodePort?

Reply:
or ServicePort?

Reply:
ya, I meant to clean up PublicServicePort but only got half way done :(

Reply:
Can we pick one and use it everywhere? Advertise Public External Service

Reply:
Changed to Advertise.